src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -484,6 +492,22 @@ def deployment_options(self):
         nvmes = 0
         res = DeploymentOptions()
         devices = {}
+
+        # setting default values
+        (res.options[OsdDeploymentOptions.COST_CAPACITY.value]
+            .option_name) = 'Cost/Capacity-optimized'
+        (res.options[OsdDeploymentOptions.THROUGHPUT.value]
+            .option_name) = 'Throughput-optimized'
+        (res.options[OsdDeploymentOptions.IOPS.value]
+            .option_name) = 'IOPS-optimized'
+
+        (res.options[OsdDeploymentOptions.COST_CAPACITY.value]
+            .msg) = 'All the available HDDs are selected'
+        (res.options[OsdDeploymentOptions.THROUGHPUT.value]
+            .msg) = 'HDDs/SSDs are selected for data devices and SSDs/NVMes for DB/WAL devices'
```

Comment:
This logic shouldn't be inside a controller (controllers should have minimal logic), even if it's a UI only one. It's better to use a separate function (it can be a method inside a class, like the factory method example I left above).

Reply:
Done!